MeetMaryam/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">   
+<link href="C:\Users\yonis\.vscode\School of Code\MeetMaryam\style.css" rel="stylesheet" type="text/css">
+
+<div class="header">
+    <h2>&#10024 Hello I'm Maryam! &#10024</h2>
+</div>

```

Comment:
It's best practice to use semantic HTML where possible.

```suggestion
<header>
    <h2>&#10024 Hello I'm Maryam! &#10024</h2>
</header>

```